recipe-client-addon/test/browser/browser_NormandyDriver.js

```diff
@@ -12,6 +12,16 @@ add_task(withDriver(Assert, async function uuids(driver) {
   isnot(uuid1, uuid2, "uuids are unique");
 }));
 
+add_task(withDriver(Assert, async function installXpi(driver) {
+  // Test that we can install an XPI from any URL
+  const dir = getChromeDir(getResolvedURI(gTestPath));
+  dir.append("fixtures");
+  dir.append("normandy.xpi");
+  const xpiUrl = Services.io.newFileURI(dir).spec;
+  var addonId = await driver.installAddon(xpiUrl);
+  await driver.uninstallAddon(addonId);
```

Comment:
The installAddon method returns the addon ID only if the installation is successful. The uninstall process would also fail if the addon ID was invalid, or if no installed addon could be found with that addonID.  Is that not sufficient or do you want this to check the list of installed addons explicitly?